config/citibank_config.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import "os"
+
+type CitibankConfig struct {
+	CitibankCertificateSSLName string
+	URLCitiBoleto              string
+	URLCiti                    string
+}
+
+func getCitibankConfig() CitibankConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀


```suggestion
func getCitibankConfig() *CitibankConfig {
```